src/Microsoft.Framework.Runtime.Roslyn/RoslynProjectReference.cs

```diff
@@ -270,7 +270,7 @@ private static bool SupportsPdbGeneration()
         {
             try
             {
-                if (PlatformHelper.IsMono)
```

Comment:
nit: I don't know if it makes a lot of difference but why this check is inside the try/catch block?